ioctl/newcmd/account/accountdelete.go

```diff
@@ -99,43 +97,53 @@ func NewAccountDelete(c ioctl.Client) *cobra.Command {
 			if err != nil {
 				return output.NewError(output.ConvertError, failToConvertStringIntoAddress, nil)
 			}
-			ks := c.NewKeyStore(config.ReadConfig.Wallet, keystore.StandardScryptN, keystore.StandardScryptP)
-			for _, v := range ks.Accounts() {
-				if bytes.Equal(account.Bytes(), v.Address.Bytes()) {
-					var confirm string
-					message := output.ConfirmationMessage{Info: infoWarn, Options: []string{"yes"}}
-					fmt.Println(message.String())
-					fmt.Scanf("%s", &confirm)
-					if !strings.EqualFold(confirm, "yes") {
-						output.PrintResult("quit")
-						return nil
-					}
 
-					if err := os.Remove(v.URL.Path); err != nil {
-						return output.NewError(output.WriteFileError, failToRemoveKeystoreFile, err)
+			var filePath string
+			if CryptoSm2 {
```

Comment:
Move CryptoSm2 into ioctl.Client

Reply:
CryptoSm2 is just set in package account.  if move into ioctl.Client, this mean other package can set it, this is not suitable, i think

Reply:
`CryptoSm2` is unnecessary to be a global var

Reply:
it is called by  ioctl/newcmd/action  and  xctl.go  (both are not added now)

Reply:
The code in newcmd can used the new `CryptoSm2` from client. And keep the old one until it isn't depended

Reply:
Move CryptoSm2 into ioctl.Client

Reply:
> is this a constant? if not, how could you prevent it be modified in other packages?

this is outdated code. Now move CryptoSm2 into ioctl.Client,  use  client.HasCryptoSm2()  to get status ,  use client.SetCryptoSm2(bool)  in main function

Reply:
> is this a constant? if not, how could you prevent it be modified in other packages?

 this is outdated code. Now move CryptoSm2 into ioctl.Client, 
 user test the result by setting corresponding gomock function.  HasCryptoSm2() to get status , SetCryptoSm2(bool) to set status